templates/service.yaml

```diff
@@ -57,7 +57,7 @@ spec:
 {{- if .Values.properties.clusterSecure }}
     - port: {{ .Values.service.httpsPort }}
       name: https
-      targetPort: {{ .Values.httpsPort }}
+      targetPort: {{ .Values.properties.httpsPort }}
```

Comment:
This change has been taken care of in another outstanding pull request - https://github.com/cetic/helm-nifi/pull/70/files#diff-1210ec3270609a5b4e28815d643b0e7b

It's different from our solution but it might also still work.